addons/website_sale/tests/test_website_sequence.py

```diff
@@ -14,7 +14,23 @@ def setUp(self):
         # if stock is installed we can't archive since there is orderpoints
         if hasattr(self.env['product.product'], 'orderpoint_ids'):
             product_templates.mapped('product_variant_ids.orderpoint_ids').write({'active': False})
-        product_templates.write({'active': False})
+
+        # Service on Timesheet can't be archived nor deleted (TASK ID 2221046)
+        # So if the product is service timesheet,
+        # we use the force arg to bypass the rule. Only valid in testing.
+        time_product = None
+        try:
+            time_product = self.env.ref('sale_timesheet_enterprise.fsm_time_product')
+        except:
+            pass # It means that we are not in enterprise, the product doesn't exist
```

Comment:
You can do this instead:
```python
time_product = self.env.ref('sale_timesheet_enterprise.fsm_time_product', raise_if_not_found=False)
```

`time_product` will either be False or contain the object :-)

Reply:
I wondered if there was a way to do that, thanks :)